libs/designer/src/lib/ui/mcp/parameters/EditOperation.tsx

```diff
@@ -222,120 +157,126 @@ export const EditOperation = forwardRef<EditOperationRef, EditOperationProps>(fu
       defaultMessage: 'Saved',
       description: 'Badge text for saved state',
     }),
-    addNewParamText: intl.formatMessage(
-      {
-        defaultMessage: 'Showing {countShowing} of {countTotal}',
-        id: 'jTHUFb',
-        description: 'Placeholder text for the number of advanced parameters showing',
-      },
-      {
-        countShowing: allConditionalSettings.length - conditionallyInvisibleSettings.length,
-        countTotal: allConditionalSettings.length,
-      }
-    ),
   };
 
-  const handleFieldChange = useCallback(
-    (field: string, value: string) => {
-      if (field === 'description') {
-        setLocalDescriptionChange(value);
-      }
+  const handleDescriptionInputChange = useCallback((description: string) => {
+    setLocalDescription(description);
+  }, []);
 
-      setIsDirty(true);
-    },
-    [setIsDirty]
-  );
+  const handleDescriptionBlur = useCallback(() => {
+    if (!selectedOperationId) {
```

Comment:
I think we can save this onSave... just keeping it in local state is fine. non blocking comment